platforms/android/tangram/src/main/java/com/mapzen/tangram/Marker.java

```diff
@@ -214,24 +214,10 @@ private boolean setBitmap(@NonNull final Bitmap bitmap) {
         final int density = context.getResources().getDisplayMetrics().densityDpi;
         final int width = bitmap.getScaledWidth(density);
         final int height = bitmap.getScaledHeight(density);
-
         final int[] argb = new int[width * height];
```

Comment:
It's also possible access the data in place in native code a9b2954f867a46348b6f1c014c871877365b5cac.